pkg/webhook/wireupwebhook.go

```diff
@@ -257,6 +260,26 @@ func setOwnerReferences(c client.Client, logger logr.Logger, deployNs string, de
 		*metav1.NewControllerRef(owner, owner.GetObjectKind().GroupVersionKind())})
 }
 
+func setWebhookOwnerReferences(c client.Client, logger logr.Logger, obj metav1.Object) {
+	channelCrdName := "channels.apps.open-cluster-management.io"
```

Comment:
Small nit comment. This can probably be referenced using a the channel schema instead of hard coded string but its not a big deal.